osgar/drivers/vesc.py

```diff
@@ -0,0 +1,114 @@
+"""
+  Driver for articulated robot Kloubak
```

Comment:
This code does not look like a driver... It is some kind of independent monitor, right?